AutoSPUpdaterModule.psm1

```diff
@@ -137,14 +137,18 @@ function Install-Remote
             {
                 $versionSwitch = "-Version 2"
             }
-            else {$versionSwitch = ""}
```

Comment:
Can we leave this the way it was just for readability/formatting purposes?

Reply:
Sure, leave it.